xml/System.IO.Enumeration/FileSystemEnumerator`1.xml

```diff
@@ -283,9 +283,9 @@
         <Parameter Name="entry" Type="System.IO.Enumeration.FileSystemEntry" RefType="ref" />
       </Parameters>
       <Docs>
-        <param name="entry">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="entry">A file system entry reference.</param>
+        <summary>Allows indicating if a file system entry should be included in the results or not.</summary>
+        <returns><see langword="true" /> if the given file system entry should be included in the results, <see langword="false" /> otherwise.</returns>
```

Comment:
```suggestion
        <returns><see langword="true" /> if the specified file system entry should be included in the results; otherwise, <see langword="false" />.</returns>
```